bin/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+set -e
+
+cargo build --target wasm32-unknown-unknown
```

Comment:
Building should never require more than this command, in the future I may want to add extra optional flags to it however so I wouldn't put it in a script.